requirements.txt

```diff
@@ -51,4 +50,4 @@ urllib3==1.25.9
 webassets==2.0
 xlrd==1.2.0
 yuicompressor==2.4.8
-lxml==4.5.1
\ No newline at end of file
+lxml==4.5.1
```

Comment:
I have no idea what this is

Reply:
@elazarg it's a whitespace change.
Perhaps windows' new line?
![Screen Shot 2020-05-25 at 17 41 50](https://user-images.githubusercontent.com/20992625/82822824-17900000-9eaf-11ea-80a7-d019e1fd1e29.png)

Reply:
I've made the change through the web interface